todo/tests/fixtures/task.py

```
@@ -1,4 +1,5 @@
 from todo.constants.task import TaskPriority
+from todo.models.task import TaskModel
```

Ruff (0.8.2)

`todo.models.task.TaskModel` imported but unused

Remove unused import: `todo.models.task.TaskModel`

(F401)